docs/kb/supporting-your-customers/adding-proxy-support-manually.md

```diff
@@ -0,0 +1,112 @@
++++
+date = "2017-08-21T00:00:00Z"
+title = "Adding Proxy Support Manually"
+description = "Instructions for adding proxy support manually"
+keywords= "proxy"
+categories = [ "Knowledgebase", "Supporting Your Customers" ]
++++
+
+The [Installation Script](/distributing-an-application/installing/#installing-behind-a-proxy) is the best way to configure Replicated to use a proxy server. This document describes the steps required to add proxy support to an existing Replicated installation outside of the installation script.
+
+### Proxy Setting Validation
+
+To validate that the proxy settings are correct and work with Replicated try using curl to hit the echo IP endpoint:
+
+```
+curl --proxy http://<proxyserveraddress> https://api.replicated.com/market/v1/echo/ip
+```
+
+If the test command fails in a proxied environment you will need to perform the steps below.
+
+## Manually Adding Proxy Server Settings
+
+The steps to setup the proxy depend on the Linux distribution and the init system:
+
+Upstart: Ubuntu 10+, Fedora 9+, RHEL6, CentOS 6, AWS Linux 2014.3.2.
+
+Systemd: Fedora 15+, RHEL7, Arch, Debian 8+, Ubuntu 15+, openSUSE 12.3+.
+
+System V: RHEL5, Debian, SLES 11.
+
+### Systemd
+
+We first need to [setup Docker](https://docs.docker.com/engine/admin/systemd/#/http-proxy) with the proxy server configuration before configuring Replicated's proxy server settings.
+
+After verifying Docker proxy settings are working correctly we will need to modify "/etc/sysconfig/replicated" and pass the variable HTTP_PROXY into REPLICATED_OPTS.
+
+```
+cat /etc/sysconfig/replicated
+RELEASE_CHANNEL=stable
+PRIVATE_ADDRESS=10.138.0.2
+SKIP_OPERATOR_INSTALL=0
+REPLICATED_OPTS="-e LOG_LEVEL=info -e HTTP_PROXY=http://<proxyserveraddress>:<proxyserverport> -e DAEMON_TOKEN=9qB38BngNIb9aiw6Aq8JCQ57t8H4yYfC -e NODENAME=mytest.node"
+systemctl daemon-reload
+```
+The install may need to turn off proxy in Replicated and Docker to talk to our local registry. If you see any errors talking to the internal registry take the IP that it is trying to talk to and update:
+
+```
+# add "-e NO_PROXY=172.17.0.1" to /etc/sysconfig/replicated at the end of REPLICATED_OPTS
```

Comment:
do you need this line? seems like  updating `/etc/systemd/system/docker.service.d/http-proxy.conf` does the same thing